blockchain/README.md

```diff
@@ -3,7 +3,7 @@ blockchain
 
 [![Build Status](https://github.com/decred/dcrd/workflows/Build%20and%20Test/badge.svg)](https://github.com/decred/dcrd/actions)
 [![ISC License](https://img.shields.io/badge/license-ISC-blue.svg)](http://copyfree.org)
-[![GoDoc](https://img.shields.io/badge/godoc-reference-blue.svg)](https://godoc.org/github.com/decred/dcrd/blockchain)
+[![Doc](https://img.shields.io/badge/doc-reference-blue.svg)](https://pkg.go.dev/github.com/decred/dcrd/blockchain)
```

Comment:
Looks like these need to include the proper version.

e.g:
https://pkg.go.dev/github.com/decred/dcrd/blockchain/v3
versus the current:
https://pkg.go.dev/github.com/decred/dcrd/blockchain